cranelift/codegen/src/isa/x64/inst/args.rs

```diff
@@ -459,6 +467,14 @@ impl SseOpcode {
             | SseOpcode::Mulpd
             | SseOpcode::Mulsd
             | SseOpcode::Orpd
+            | SseOpcode::Psllw
+            | SseOpcode::Pslld
+            | SseOpcode::Psllq
```

Comment:
The way to go here would be to use the `OperandSize` (maybe add variants for lower sizes), and have it in the instruction, instead of having one SSE opcode per size. Since you need a new Inst anyways, how about doing this here?

Reply:
I had something like the `OperandSize` approach but based on someone's comments, I refactored it to this.